retworkx-core/src/shortest_path/dijkstra.rs

```diff
@@ -98,24 +138,25 @@ use crate::min_scored::MinScored;
 /// assert_eq!(res.unwrap(), expected_res);
 /// // z is not inside res because there is not path from b to z.
 /// ```
-pub fn dijkstra<G, F, K, E>(
+pub fn dijkstra<G, F, K, E, S>(
```

Comment:
Can we mention in the docs that you can choose the return type for this function? It currently states that it returns a `DictMap`.